osu.Game/Screens/Play/GameplayMenuOverlay.cs

```diff
@@ -122,13 +122,15 @@ private void load(OsuColour colours)
                 },
             };
 
-            Retries = 0;
+            Retries = retries;
         }
 
+        private int retries;
         public int Retries
         {
             set
             {
+                retries = value;

```

Comment:
Generally, properties should start with an equality check, in this case
`if (value == retries) return`

I think you should do this here and then create follow it with:

```
retries = value;
if (retryCounterContainer != null)
    updateRetryCount();
```

then change the code in `load()` to call `updateRetryCount()`